debian/hwangsae-recorder.install

```diff
@@ -1,2 +1,2 @@
-usr/bin/hwangsae-recorder-agent
+usr/bin/hwangsae-simple-recorder-agent
```

Comment:
Hmm, I'd install both versions for now.

Reply:
But there is just one executable agent version `hwangsae-simple-recorder-agent` which is the simplest and easiest implementation, is supports only one simultaneous recording and it based on `hwangsae_recorder_agent`.

`hwangsae_recorder_agent` is a base recorder agent which implements the common code, such as DBUS API, Lookup, Url, etc. but does not implement `start` nor `stop`

Based on this work I'll later push `hwangsae-mulit-recorder-agent` which will be a new agent which support multiple recording simultaneously, of course base on the common code in `hwangsae_recorder_agent`

Reply:
Ah, I see. Yes that makes sense.